File: web3rpc/rpc-specs/code-samples/python/klay/transaction/getDecodedAnchoringTransactionByHash.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

hashOfTransaction = "0x72cde80650c7c6745e4cf4c162e9ce1e5542f4d86112925faa6aa75f6a6142ec"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_decoded_anchoring_transaction_by_hash(hashOfTransaction)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/transaction/getDecodedAnchoringTransactionByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const hashOfTransaction = '0x026b64e16b86633c0199f78f37a64840d3601d83e5c799f115b63024764524ca'

    sdk.klay.getDecodedAnchoringTransactionByHash(hashOfTransaction, {}, (err, data, response) => {
        console.log(data);
    });
}
)()